nodeup/pkg/model/docker.go

```diff
@@ -571,6 +612,28 @@ var dockerVersions = []dockerVersion{
 		Hash:          "0a1325e570c5e54111a79623c9fd0c0c714d3a11",
 		Dependencies:  []string{"libtool-ltdl", "libseccomp", "libcgroup"},
 	},
+
+	// 18.06.2 - CentOS / Rhel7 (two packages)
+	{
+		DockerVersion: "18.06.2",
+		Name:          "container-selinux-2",
```

Comment:
The name of the package is actually `container-selinux`. If you let it as it is `kops-configuration` will keep retrying to install and will fail causing the Node to be lost.

Reply:
Fixed in #6492.  We actually get away with it because we don't double-check the package installed what we expected, but I started to fix it up in #6490